python-sdk/tests/telemetry/test_tracing.py

```diff
@@ -0,0 +1,50 @@
+# NOTE: This test will result in logs to the console about a missing LangWatch API key,
+# which is expected as we don't care about integration-level API reporting in this test.
```

Comment:
@0xdeafcafe can't you mock/replace the logging just for this test so that it doesn't log, instead of having a note like this?

Reply:
I gave up trying to do this lol